test/utils/helpers.test.ts

```diff
@@ -20,3 +18,25 @@ it('evenHex', () => {
   expect(evenHex('a')).to.eql('0a');
   expect(evenHex('0x')).to.eql('');
 });
+
+describe('isTimestampMoreRecentThan', () => {
+  const timestamp = Math.floor(Date.now() / 1000);
+
+  describe('when timestamp is greater the current time', () => {
+    it('returns false', () => {
+      expect(isTimestampMoreRecentThan(timestamp + 1000, 60)).to.be.false;
+    });
+  });
```

Comment:
... greater than ...

We set timestamp + 1000 to see if it is more recent than 60 second. Are we looking 1000 seconds in the future and it is not more recent than 60 seconds? I don't get it.

Reply:
Looking at the implementation, the timestamp cannot be greater than the current date.